### ISSUE TITLE
Demo is not working

### ISSUE THREAD

**Issue by shershen08** [manuelbieh/Geolib#133]
The url stated in the README as Demo - https://www.manuelbieh.de/publikationen/scripts/geolib/demo.html
Returns 404

**Comment by manuelbieh** [comment]
Yes. It broke when I relaunched my website a couple of days ago. Thanks for noticing. I'll fix it soon :wink:

**Comment by nformica** [comment]
I tried the demo link, but it's down again !

**Comment by carneirocorp** [comment]
Still not OK
